app/packs/src/onchain/utils.js

```diff
@@ -7,3 +8,14 @@ export const parseAndCommify = (value, nrOfDecimals = 2) => {
 export const ipfsToURL = (ipfsAddress) => {
   return "https://ipfs.io/" + ipfsAddress.replace("://", "/");
 };
+
+export const getENSFromAddress = async (walletAddress) => {
+  let name = null;
+  try {
+    const provider = new ethers.providers.EtherscanProvider();
+    const name = await provider.lookupAddress(walletAddress);
```

Comment:
Constant already defined above

```suggestion
    name = await provider.lookupAddress(walletAddress);
```